service/history/queues/scheduler.go

```diff
@@ -28,10 +28,17 @@ package queues
 
 import (
 	"go.temporal.io/server/common"
+	"go.temporal.io/server/common/dynamicconfig"
 	"go.temporal.io/server/common/log"
 	"go.temporal.io/server/common/log/tag"
 	"go.temporal.io/server/common/metrics"
+	"go.temporal.io/server/common/namespace"
 	"go.temporal.io/server/common/tasks"
+	"go.temporal.io/server/service/history/configs"
+)
+
+const (
+	defaultParallelProcessorQueueSize = 10
```

Comment:
add comment, what is this used for (per shard? per host? per queue type? per namespace? per priority?)